packages/rollup-full-node/src/types/handler.ts

```diff
@@ -37,4 +37,9 @@ export enum Web3RpcMethods {
   getTransactionReceipt = 'eth_getTransactionReceipt',
   networkVersion = 'net_version',
   sendRawTransaction = 'eth_sendRawTransaction',
+
+  // Test methods:
+  setTimestamp = 'test_setTimestamp',
```

Comment:
One thing to note is that we could consider making the methods match more closely ganache's custom RPC methods wrt timestamp -- https://github.com/trufflesuite/ganache-core#custom-methods

Reply:
In Ganache they have `evm_increaseTime` and `evm_mine` which can optionally accept a new timestamp

Reply:
Switched this to `evm_increaseTime` so that time can only be added, never removed. Punting on `evm_mine`.